lib/spack/spack/installer.py

```diff
@@ -1663,6 +1685,10 @@ def __init__(self, pkg, compiler, start, attempts, status, installed):
         self.dependents = set(package_id(d.package) for d
                               in self.spec.dependents())
 
+        tty.debug('Pkg id {0} has the following dependents:'.format(self.pkg_id))
```

Comment:
You probably want to remove this, it's a leftover debugging statement.

Reply:
Indeed, thanks @haampie, I plan to get a little feedback on the approach in general, make sure change will be acceptable, then I'll make a cleanup pass and fix the 15 or so broken tests.